client/my-sites/site-settings/composing/style.scss

```diff
@@ -0,0 +1,7 @@
+.composing__module-settings.is-indented {
+	margin: 16px 32px 0;
```

Comment:
Can we drop this in favor of the solution offered here: https://github.com/Automattic/wp-calypso/pull/11240